### PR TITLE
Landing/Jobs listing page

### PR DESCRIPTION
**Pull request by daemyonc**
I'll start working on landing-jobs listing page

### CODE REVIEW

**Comment by daemyonc** [comment]
TopNav1
Where I am at the moment

**Comment by ShaunWMusic** [comment]
nice dude! progress! if you want this PR to just focus on TopNav. that'd work! that way, i can pull that in sooner instead of waiting for full pr to be done

**Comment by daemyonc** [comment]
Thanks. Yeah, we can just let it be for TopNav. I won't rename it unless you want me to but the others can named after each section/component. I'll get the styling done and then we can mark it done? Routes can be there own PR?

**Comment by ShaunWMusic** [comment]
perfect. and update screen shoot when finished🥳

File: tests/integration/components/top-nav/component-test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    // Template block usage:
+    await render(hbs`
+      <TopNav>
+        template block text
+      </TopNav>
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
+  });
+});
```
Comment (ShaunWMusic) [inline tests/integration/components/top-nav/component-test.js:26]: this is one of the benefites of ember, is that it generate a generic test code file for you that simply tests the components, routes, etc... you can also add to this test to ensure specific UI remains in the code... or create unit tests to ensure functionality doesn't change.

Comment (daemyonc) [reply tests/integration/components/top-nav/component-test.js:26]: Okay cool. Yeah, I need to learn testing techniques. Started a little bit from the docs.

**Comment by daemyonc** [comment]
TopNav1
TopNav-hover
That menu hover effect was a booger.

**Review by puglisac — APPROVED** [review]
Man, this looks great!

**Comment by ShaunWMusic** [comment]
LGTM

**Pull request merged**